### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by faloi**
:wave:! GitHub Classroom created this pull request as a place for your teacher to leave feedback on your work. It will update automatically. **Don’t close or merge this pull request**, unless you’re instructed to do so by your teacher.

In this pull request, your teacher can leave comments and feedback on your code. Click the **Subscribe** button to be notified if that happens.

Click the **Files changed** or **Commits** tab to see all of the changes pushed to `master` since the assignment started. Your teacher can see this too.

<details>
<summary>
<strong>Notes for teachers</strong>
</summary>

Use this PR to leave feedback. Here are some tips:

- Click the **Files changed** tab to see all of the changes pushed to `master` since the assignment started. To leave comments on specific lines of code, put your cursor over a line of code and click the blue **+** (plus sign). To learn more about comments, read “[Commenting on a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request)”.
- Click the **Commits** tab to see the commits pushed to `master`. Click a commit to see specific changes.
- If you turned on autograding, then click the **Checks** tab to see the results.
- This page is an overview. It shows commits, line comments, and general comments. You can leave a general comment below.

For more information about this pull request, read “[Leaving assignment feedback in GitHub](https://classroom.github.com/help/leaving-feedback-in-github)”.
</details>

Subscribed: @AgusFiorda

### CODE REVIEW

**Comment by AgusFiorda** [comment]
@faloi termine ! 👍

File: README.md [hunk 1]
```diff
@@ -22,4 +22,25 @@ Y de yapa, dos ejemplos de cómo incluir una imagen en Markdown (vean este archi
...
+**Diagrama de Plagas**
+
+![alt text](https://github.com/obj2-unahur-2020s2/documentando-wollok-AgusFiorda/blob/master/plagas.png)
+
+**Diagrama de Elementos**
+![alt text](https://github.com/obj2-unahur-2020s2/documentando-wollok-AgusFiorda/blob/master/elementos.png)
+
+**Diagrama de Barrio**
+
+![alt text](https://github.com/obj2-unahur-2020s2/documentando-wollok-AgusFiorda/blob/master/barrio.png)
```
Comment (faloi) [inline README.md:41]: Se suponía que el `Algo` y `OtroAlgo` que yo puse tenías que cambiarlo por los tipos de verdad. :sweat_smile:

File: src/barrio.wlk [hunk 2]
```diff
@@ -1,21 +1,23 @@
 class Barrio {
-
+	//tipo: lista
```
Comment (faloi) [inline src/barrio.wlk:2]: Ok, pero incompleto: _¿lista de qué?_.

File: src/barrio.wlk [hunk 3]
```diff
@@ -1,21 +1,23 @@
 class Barrio {
-
+	//tipo: lista 
 	const property elementos = []
-
+	
+	//agrega un elemento a la lista, no devuelve nada
```
Comment (faloi) [inline src/barrio.wlk:5]: Cuando pensamos en el tipo solo hablamos de eso, no importa qué hace el método.

File: src/barrio.wlk [hunk 4]
```diff
@@ -1,21 +1,23 @@
...
 	method agregarElemento(elemento) {
 		elementos.add(elemento)
 	}
-
+	// quita un elemento de la lista, no devuelve nada
 	method sacarElemento(elemento) {
 		elementos.remove(elemento)
 	}
-
+	//busca los elementosBuenos de la lista elementos y los devuelve.
```
Comment (faloi) [inline src/barrio.wlk:13]: Mal, no devuelve los elementos, devuelve un **número**.

File: src/barrio.wlk [hunk 5]
```diff
@@ -1,21 +1,23 @@
...
 	method elementosBuenos() {
 		return elementos.count{ e => e.esBueno() }
 	}
-
+	// cuenta la cantidad de elementos que no son buenos. devuelve el numero de elementos malos
 	method elementosMalos() = elementos.count{ e => not e.esBueno() }
-
+	
+	//devuelve un true o false, dependiendo esa condicion
```
Comment (faloi) [inline src/barrio.wlk:20]: Bien, aunque ese tipo se llama **booleano**.

File: src/elementos.wlk [hunk 6]
```diff
@@ -1,40 +1,48 @@
...
 
 }
 
 class Mascota {
-
+	//tipo de dato: entero
 	var property nivelSalud
-
+	
+	//no devuelve nada, le asigna un valor a nivelSalud
```
Comment (faloi) [inline src/elementos.wlk:25]: Falta el tipo del parámetro `plaga`.

File: src/elementos.wlk [hunk 7]
```diff
@@ -1,40 +1,48 @@
...
 	method fueAtacado(plaga) {
 		if (plaga.transmiteEnfermedad()) {
 			nivelSalud -= plaga.nivelDanio()
 		}
 	}
-
+	//no devuelve nada, le asigna un valor a nivelSalud
 	method esBueno() = nivelSalud > 250
```
Comment (faloi) [inline src/elementos.wlk:32]: Mal, sí que devuelve. El `=` es una forma de definir métodos que devuelven algo en una sola línea.

**Pull request closed without merging**